dev/io.openliberty.microprofile.telemetry.1.0.internal_fat/fat/src/io/openliberty/microprofile/telemetry/internal_fat/TelemetryLongRunningTest.java

```diff
@@ -54,10 +52,30 @@ public static void setUp() throws Exception {
                         .addPackage(LongRunningTask.class.getPackage())
                         .addPackage(InMemorySpanExporter.class.getPackage())
                         .addAsServiceProvider(ConfigurableSpanExporterProvider.class, InMemorySpanExporterProvider.class)
-                        .addAsResource(new StringAsset("otel.sdk.disabled=false\notel.traces.exporter=in-memory\notel.bsp.schedule.delay=100"),
+                        .addAsResource(new StringAsset("otel.sdk.disabled=false\notel.traces.exporter=in-memory"),
                                        "META-INF/microprofile-config.properties");
 
         ShrinkHelper.exportAppToServer(server, app, SERVER_ONLY);
         server.startServer();
     }
+
+    @Test
+    public void testConsole() throws Exception {
+        server.setMarkToEndOfLog();
+
+        int timeout = DEFAULT_TIMEOUT;
+        while (timeout > 0) {
```

Comment:
```suggestion
        long timeoutNanos = DEFAULT_TIMEOUT.toNanos();
        long startNanoTime = System.nanoTime();
        while (System.nanoTime() - startNanoTime < timeoutNanos) {
```

We want to measure the actual time elapsed, rather than counting how long we've slept for since just counting sleep time doesn't take account of how long the request may have taken (which, if we've made a mistake or there's an infrastructure issue, can be longer than we expect).

To measure durations, use `System.nanoTime()` and use the `Duration` class to do any conversions required.

Note: read the `System.nanoTime()` javadoc carefully, it can wrap around and can be negative but you can avoid all the pitfalls by always subtracting the end time from the start time to get a duration in nanoseconds.

If you prefer, you can do everything with `Duration`s here, but it gets a bit messy:
```java
        long startNanoTime = System.nanoTime();
        while (Duration.ofNanos(System.nanoTime() - startNanoTime).compareTo(DEFAULT_TIMEOUT) < 0) {
```
As well as being longer, having to call `compareTo` directly is harder to read.